README.md

```diff
@@ -8,11 +8,11 @@ places they need to be for Rancher and its associated projects to use them.
 `regsync` is used for mirroring images from one repository to another. `regsync`
 is configured via `regsync.yaml`. `regsync.yaml` is generated by running the
```

Comment:
We should have a little blurb that says something like "this used to be image-mirror; it was renamed to artifact-mirror because... see issue X for more details". Maybe use something like in https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts?